### PR TITLE
Landing page analytics

### PR DESCRIPTION
**Pull request by fitzgeraldkd**
Minor analytics updates for the new landing pages.
Also fixed a bug where Sentry messages were not being captured.

### CODE REVIEW

File: src/analytics.ts [hunk 1]
```diff
@@ -171,15 +174,9 @@ function getEventProperties(eventName: string, target: HTMLElement) {
 /**
  * It is possible for browsers to block the Sentry script from being downloaded, so capture messages safely.
  */
-function safelyCaptureMessage(message: string, level: SeverityLevel = null, context: SentryContext = null) {
+function safelyCaptureMessage(message: string, level: SeverityLevel = null) {
     if (typeof Sentry !== 'undefined') {
-        Sentry.withScope(function (scope) {
```
Comment (fitzgeraldkd) [inline src/analytics.ts:176]: Nothing in the callback was running, so events weren't being sent. We only used the scope for the A/B testing, which is currently inactive, so I've just disabled the context for now and added a TODO in the Google Analytics block.

File: dist/analytics.js [hunk 2]
```diff
@@ -58,7 +58,8 @@ function getGoogleAnalyticsProperties() {
                 return { experiment_group: experimentGroup, experiment_id: experimentId };
             }
         }
-        safelyCaptureMessage('The Google Optimize experiment group could not be determined.', 'warning', { properties: { experimentCookie: experimentCookie } });
+        // TODO: Investigate how to properly attach a context to Sentry messages and include the experiment cookie.
+        safelyCaptureMessage('The Google Optimize experiment group could not be determined.', 'warning');
```
Comment (martingouy) [inline dist/analytics.js:62]: 👍

File: src/analytics.ts (hunk 1 above)
Comment (martingouy) [reply src/analytics.ts:176]: That's reasonable!

**Review by martingouy — APPROVED** [review]
LGTM!

**Pull request merged**